MPower/app/src/main/res/values/strings.xml

```diff
@@ -15,4 +15,23 @@
         Download it for Android at https://play.google.com/store/apps/details?id=org.sagebase.mpower</string>
     <string name="rsb_share_app_url">https://play.google.com/store/apps/details?id=org.sagebase.mpower</string>
 
+    <!-- Custom Mood Survey -->
+    <string name="custom_mood_survey_title">Custom Survey</string>
+    <string name="custom_mood_survey_text">You now have the ability to create your own survey question. Tap Next to enter your question.</string>
+    <string name="custom_mood_survey_placeholder_text">Tap to write</string>
+
+    <string name="custom_mood_survey_question_text">Customize your question.</string>
+
+    <string name="custom_mood_survey_dialog_label">Learn More</string>
+    <string name="custom_mood_survey_dialog_text">Creating a custom question will help you track something personal to you over time. Think about something you care deeply about and would like to see how your performance in that area changes with your post-treatment evolution.
+Some examples may include:
+- How was your morning run today?
+- My work day today was...
+
+We will track your question in the dashboard (shown as "Custom Question") over time. Remember that you can always go to your profile and change this question.</string>
```

Comment:
"Remember that you can always go to your profile and change this question."  Is there some sort of UI that needs to be shown in the profile for this?  Can you grab a screenshot?